_component_design/chips.md

```diff
@@ -52,16 +112,16 @@ Chips are best used with indicating a list of items that may be part of a differ
 
 {% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
 
-### Default chip
+### Spacing
 
-The default chip is a single circle that contains a preview of the item the chip represents. The preview can consist of the the following; letters to indicate the name, an image to indicate a user, or a product icon to indicate the type of device.
+Use the following spacing guidelines when you construct chips.
```

Comment:
s/`Use the following spacing guidelines`/`Use these spacing guidelines` (same reason as above)